apple-pie/index.html

```diff
@@ -3,37 +3,88 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles/style.css">
     <title>Apple Pie Recipe</title>
     <!-- Remember to link your styles -->
   </head>
   <body>
-    Apple Pie
 
-    This was my grandmother's apple pie recipe. I have never seen another one quite like it. It will always
-    be my favorite and has won me several first place prizes in local competitions. I hope it becomes one of your
-    favorites as well! 
     
-    Ingredients 
+    <section id="head" class="nomargin">  
+
+      <div class="image">      
+        <h1> Apple Pie </h1>
+      </div>
+      
```

Comment:
just have to check the identation probably you want to add vscode prettier extension make a lot of work for you when hit the save action